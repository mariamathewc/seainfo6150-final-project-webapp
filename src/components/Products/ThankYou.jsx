import React from "react";
import styles from "./ThankYou.module.css";


const ThankYou = (props) => {

 

    return (

        <div className={styles.row}>
            <div className={styles.texts}>
                <h1><em>Thank you for shopping at Travel and Tents</em></h1>
                </div>
            
        </div>



    );


}

export default ThankYou