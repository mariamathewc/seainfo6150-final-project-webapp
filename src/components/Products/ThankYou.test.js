import React from "react";
import ThankYou from "./ThankYou.jsx";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';




describe("ThankYou test", () => {
	it("renders correctly", () => {
		const { container } = render(<BrowserRouter><ThankYou /></BrowserRouter>);
		expect(container).toMatchSnapshot();
	});
});