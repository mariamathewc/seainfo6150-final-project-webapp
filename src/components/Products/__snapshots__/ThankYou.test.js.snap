// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThankYou test renders correctly 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="texts"
    >
      <h1>
        <em>
          Thank you for shopping at Travel and Tents
        </em>
      </h1>
    </div>
  </div>
</div>
`;
